_topic_map.yml

```diff
@@ -302,8 +302,22 @@ Name: roxctl CLI
 Dir: cli
 Distros: openshift-acs
 Topics:
-- Name: Getting started with the roxctl CLI
-  File: getting-started-cli
+- Name: Installing the roxctl CLI
```

Comment:
(applies to all names): Do we require `roxctl CLI` always in the name, or can we live with `roxctl`?

Personally would drop the `CLI` but not sure whether its required from a naming clearness / fits how other products document their CLIs.

Reply:
I think we have been using all across ACS documentation, so IMHO I think we can keep it the way it is. 
@kcarmichael08 WDYT?